lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -629,8 +631,9 @@ def _section_send_email(course, access):
 def _get_dashboard_link(course_key):
     """ Construct a URL to the external analytics dashboard """
     analytics_dashboard_url = '{0}/courses/{1}'.format(settings.ANALYTICS_DASHBOARD_URL, unicode(course_key))
-    link = u"<a href=\"{0}\" target=\"_blank\">{1}</a>".format(analytics_dashboard_url,
-                                                               settings.ANALYTICS_DASHBOARD_NAME)
+    link = HTML(u"<a href=\"{0}\" target=\"_blank\">{1}</a>").format(
```

Comment:
I realize that the original change is to safe template it, so this is totally a NIT and optional: I think this would look a bit cleaner by using single quote to wrap: `HTML(u'<a href="{0}" target="_blank">{1}</a>')`.